pure.zsh

```diff
@@ -47,6 +47,15 @@ prompt_pure_check_cmd_exec_time() {
 	(($elapsed > ${PURE_CMD_MAX_EXEC_TIME:=5})) && prompt_pure_human_time $elapsed
 }
 
+prompt_pure_clear_screen() {
+	# enable output to terminal
+	zle -I
+	# clear screen and move cursor to (0, 0)
+	print -n "\e[2J\e[0;0H"
```

Comment:
Single-quotes would work, no doubt. Although, when working with shell scripts I kind of prefer double quotes. It's easier to add a variable later (no need to convert quotes) and you don't really need to think about which to use. The exception I usually do is when you have double-quotes inside the string, then I might wrap it in single-quotes.

I'm fine with either as long as we stick to one way of doing it :)!